### PR TITLE
[OS-159] (Carnation) Broken Schedules Experience

### PR DESCRIPTION
**Pull request by aaronjchristian**
## Steps for review

- [ ] Enable the OpenY "Carnation" theme. 
- [ ] Visit any of the new schedules pages. (/swim)
- [ ] Ensure schedules works properly using filter sets etc.
- [ ] Review "Toggle Buttons" as they weren't functioning properly.

https://www.drupal.org/project/openy/issues/2999141#comment-12770901

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer CI. <br/>CI code sniffer errors
- [ ] All tests are running and there are no failed tests reported by CI. <br/>Behat test results
- [x] [Documentation](https://github.com/ymcatwincities/openy/tree/8.x-1.x/docs) has been updated according to PR changes.
- [x] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [x] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md).
- [x] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [x] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

File: themes/openy_themes/openy_carnation/templates/openy_repeat/openy-repeat-schedule-dashboard--sidebar.html.twig [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+              <div class="checkbox-wrapper">
+                <input type="checkbox" v-model="categories" v-bind:class="{ hidden: categoryExcluded('{{ category }}') }" value="{{ category }}" id="cb-{{ category|lower|replace(' ', '-') }}" class="box" {{ cat_checked }}>
+                <label for="cb-{{ category|lower|replace(' ', '-') }}">{{ category }}</label>
+              </div>
+            {% endfor %}
+          </div>
+        </div>
+      {% endif %}
+
+      {# @todo: class name (implement) #}
```
Comment (podarok) [inline themes/openy_themes/openy_carnation/templates/openy_repeat/openy-repeat-schedule-dashboard--sidebar.html.twig]: can we remove commented code?

**Review by podarok — CHANGES_REQUESTED** [review]
Small change to get merged

**Comment by podarok** [comment]
ok to test

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build979 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build979 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build979 |
| Installation process | http://install.openy.cibox.tools/build979/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/979/

**Comment by podarok** [comment]
retest this please

**Comment by podarok** [comment]
@aaronjchristian are you sure this is planned for November?
If this is a bug fix - change git base to 8.x-1.x , please

**Comment by bjwilkins** [comment]
!!! Couldn't read comment file !!!
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/984/

**Build Log**
*last 1000 lines*

 ```
GitHub pull request #1295 of commit dedf057cd1a708b7efbd9604c4358eb18986ce8c automatically merged.
Building in workspace /var/lib/jenkins/jobs/PR_BUILDER_COMPOSER/workspace
Cloning the remote Git repository
Using shallow clone
Cloning repository https://github.com/ymcatwincities/openy.git
 > git init /var/lib/jenkins/jobs/PR_BUILDER_COMPOSER/workspace/build984/openy-profile # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
 > git --version # timeout=10
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git6296069313655009356.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/heads/*:refs/remotes/origin/* --depth=1 # timeout=5
 > git config --local --remove-section credential # timeout=10
ERROR: Error cloning remote repo 'origin'
hudson.plugins.git.GitException: Command "git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/heads/*:refs/remotes/origin/* --depth=1" returned status code 128:
stdout: 
stderr: remote: Counting objects: 9441, done.
remote: Compressing objects:   0% (1/5764)   
remote: Compressing objects:   1% (58/5764)   
remote: Compressing objects:   2% (116/5764)   
remote: Compressing objects:   3% (173/5764)   
remote: Compressing objects:   4% (231/5764)   
remote: Compressing objects:   5% (289/5764)   
remote: Compressing objects:   6% (346/5764)   
remote: Compressing objects:   7% (404/5764)   
remote: Compressing objects:   8% (462/5764)   
remote: Compressing objects:   9% (519/5764)   
remote: Compressing objects:  10% (577/5764)   
remote: Compressing objects:  11% (635/5764)   
remote: Compressing objects:  12% (692/5764)   
remote: Compressing objects:  13% (750/5764)   
remote: Compressing objects:  14% (807/5764)   
remote: Compressing objects:  15% (865/5764)   
remote: Compressing objects:  16% (923/5764)   
remote: Compressing objects:  17% (980/5764)   
remote: Compressing objects:  18% (1038/5764)   
remote: Compressing objects:  19% (1096/5764)   
remote: Compressing objects:  20% (1153/5764)   
remote: Compressing objects:  21% (1211/5764)   
remote: Compressing objects:  22% (1269/5764)   
remote: Compressing objects:  23% (1326/5764)   
remote: Compressing objects:  24% (1384/5764)   
remote: Compressing objects:  25% (1441/5764)   
remote: Compressing objects:  26% (1499/5764)   
remote: Compressing objects:  27% (1557/5764)   
remote: Compressing objects:  28% (1614/5764)   
remote: Compressing objects:  29% (1672/5764)   
remote: Compressing objects:  30% (1730/5764)   
remote: Compressing objects:  31% (1787/5764)   
remote: Compressing objects:  32% (1845/5764)   
remote: Compressing objects:  33% (1903/5764)   
remote: Compressing objects:  34% (1960/5764)   
remote: Compressing objects:  35% (2018/5764)   
remote: Compressing objects:  36% (2076/5764)   
remote: Compressing objects:  37% (2133/5764)   
remote: Compressing objects:  38% (2191/5764)   
remote: Compressing objects:  39% (2248/5764)   
remote: Compressing objects:  40% (2306/5764)   
remote: Compressing objects:  41% (2364/5764)   
remote: Compressing objects:  42% (2421/5764)   
remote: Compressing objects:  43% (2479/5764)   
remote: Compressing objects:  44% (2537/5764)   
remote: Compressing objects:  45% (2594/5764)   
remote: Compressing objects:  46% (2652/5764)   
remote: Compressing objects:  47% (2710/5764)   
remote: Compressing objects:  48% (2767/5764)   
remote: Compressing objects:  49% (2825/5764)   
remote: Compressing objects:  50% (2882/5764)   
remote: Compressing objects:  51% (2940/5764)   
remote: Compressing objects:  52% (2998/5764)   
remote: Compressing objects:  53% (3055/5764)   
remote: Compressing objects:  54% (3113/5764)   
remote: Compressing objects:  55% (3171/5764)   
remote: Compressing objects:  56% (3228/5764)   
remote: Compressing objects:  57% (3286/5764)   
remote: Compressing objects:  58% (3344/5764)   
remote: Compressing objects:  59% (3401/5764)   
remote: Compressing objects:  60% (3459/5764)   
remote: Compressing objects:  61% (3517/5764)   
remote: Compressing objects:  62% (3574/5764)   
remote: Compressing objects:  63% (3632/5764)   
remote: Compressing objects:  64% (3689/5764)   
remote: Compressing objects:  65% (3747/5764)   
remote: Compressing objects:  66% (3805/5764)   
remote: Compressing objects:  67% (3862/5764)   
remote: Compressing objects:  68% (3920/5764)   
remote: Compressing objects:  69% (3978/5764)   
remote: Compressing objects:  70% (4035/5764)   
remote: Compressing objects:  71% (4093/5764)   
remote: Compressing objects:  72% (4151/5764)   
remote: Compressing objects:  73% (4208/5764)   
remote: Compressing objects:  74% (4266/5764)   
remote: Compressing objects:  75% (4323/5764)   
remote: Compressing objects:  76% (4381/5764)   
remote: Compressing objects:  77% (4439/5764)   
remote: Compressing objects:  78% (4496/5764)   
remote: Compressing objects:  79% (4554/5764)   
remote: Compressing objects:  80% (4612/5764)   
remote: Compressing objects:  81% (4669/5764)   
remote: Compressing objects:  82% (4727/5764)   
remote: Compressing objects:  83% (4785/5764)   
remote: Compressing objects:  84% (4842/5764)   
remote: Compressing objects:  85% (4900/5764)   
remote: Compressing objects:  86% (4958/5764)   
remote: Compressing objects:  87% (5015/5764)   
remote: Compressing objects:  88% (5073/5764)   
remote: Compressing objects:  89% (5130/5764)   
remote: Compressing objects:  90% (5188/5764)   
remote: Compressing objects:  91% (5246/5764)   
remote: Compressing objects:  92% (5303/5764)   
remote: Compressing objects:  93% (5361/5764)   
remote: Compressing objects:  94% (5419/5764)   
remote: Compressing objects:  95% (5476/5764)   
remote: Compressing objects:  96% (5534/5764)   
remote: Compressing objects:  97% (5592/5764)   
remote: Compressing objects:  98% (5649/5764)   
remote: Compressing objects:  99% (5707/5764)   
remote: Compressing objects: 100% (5764/5764)   
remote: Compressing objects: 100% (5764/5764), done.
Receiving objects:   0% (1/9441)   
Receiving objects:   1% (95/9441)   
Receiving objects:   2% (189/9441)   
Receiving objects:   3% (284/9441)   
Receiving objects:   4% (378/9441)   
Receiving objects:   5% (473/9441)   
Receiving objects:   6% (567/9441)   
Receiving objects:   7% (661/9441)   
Receiving objects:   8% (756/9441)   
Receiving objects:   9% (850/9441)   
Receiving objects:  10% (945/9441)   
Receiving objects:  11% (1039/9441)   
Receiving objects:  12% (1133/9441)   
Receiving objects:  13% (1228/9441)   
Receiving objects:  14% (1322/9441)   
Receiving objects:  15% (1417/9441)   
Receiving objects:  16% (1511/9441)   
Receiving objects:  17% (1605/9441)   
Receiving objects:  18% (1700/9441)   
Receiving objects:  19% (1794/9441)   
Receiving objects:  20% (1889/9441)   
Receiving objects:  21% (1983/9441)   
Receiving objects:  22% (2078/9441)   
Receiving objects:  23% (2172/9441)   
Receiving objects:  24% (2266/9441)   
Receiving objects:  25% (2361/9441)   
Receiving objects:  26% (2455/9441)   
Receiving objects:  27% (2550/9441)   
Receiving objects:  28% (2644/9441)   
Receiving objects:  29% (2738/9441)   
Receiving objects:  30% (2833/9441)   
Receiving objects:  31% (2927/9441)   
Receiving objects:  32% (3022/9441)   
Receiving objects:  33% (3116/9441)   
Receiving objects:  34% (3210/9441)   
Receiving objects:  35% (3305/9441)   
Receiving objects:  36% (3399/9441)   
Receiving objects:  37% (3494/9441)   
Receiving objects:  38% (3588/9441)   
Receiving objects:  39% (3682/9441)   
Receiving objects:  40% (3777/9441)   
Receiving objects:  41% (3871/9441)   
Receiving objects:  42% (3966/9441)   
Receiving objects:  43% (4060/9441)   
Receiving objects:  44% (4155/9441)   
Receiving objects:  45% (4249/9441)   
Receiving objects:  46% (4343/9441)   
Receiving objects:  47% (4438/9441)   
Receiving objects:  48% (4532/9441)   
Receiving objects:  49% (4627/9441)   
Receiving objects:  50% (4721/9441)   
Receiving objects:  51% (4815/9441)   
Receiving objects:  52% (4910/9441)   
Receiving objects:  53% (5004/9441), 8.38 MiB | 16.72 MiB/s   
Receiving objects:  53% (5050/9441), 21.43 MiB | 21.39 MiB/s   
Receiving objects:  54% (5099/9441), 21.43 MiB | 21.39 MiB/s   
fatal: write error: No space left on device
fatal: index-pack failed

	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl.launchCommandIn(CliGitAPIImpl.java:1600)
	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl.launchCommandWithCredentials(CliGitAPIImpl.java:1363)
	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl.access$300(CliGitAPIImpl.java:61)
	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl$1.execute(CliGitAPIImpl.java:299)
	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl$2.execute(CliGitAPIImpl.java:485)
	at hudson.plugins.git.GitSCM.retrieveChanges(GitSCM.java:1003)
	at hudson.plugins.git.GitSCM.checkout(GitSCM.java:1043)
	at hudson.scm.SCM.checkout(SCM.java:485)
	at hudson.model.AbstractProject.checkout(AbstractProject.java:1277)
	at hudson.model.AbstractBuild$AbstractBuildExecution.defaultCheckout(AbstractBuild.java:610)
	at jenkins.scm.SCMCheckoutStrategy.checkout(SCMCheckoutStrategy.java:86)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:532)
	at hudson.model.Run.execute(Run.java:1741)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:381)
ERROR: null
Couldn't read comment file
java.io.FileNotFoundException: File '/var/lib/jenkins/jobs/PR_BUILDER_COMPOSER/workspace/build984/commentinfo.md' does not exist
	at org.apache.commons.io.FileUtils.openInputStream(FileUtils.java:299)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1711)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1748)
	at org.jenkinsci.plugins.ghprb.GhprbBuilds.onCompleted(GhprbBuilds.java:124)
	at org.jenkinsci.plugins.ghprb.GhprbBuildListener.onCompleted(GhprbBuildListener.java:27)
	at org.jenkinsci.plugins.ghprb.GhprbBuildListener.onCompleted(GhprbBuildListener.java:12)
	at hudson.model.listeners.RunListener.fireCompleted(RunListener.java:201)
	at hudson.model.Run.execute(Run.java:1786)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:381)
```

**Comment by podarok** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build990 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build990 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build990 |
| Installation process | http://install.openy.cibox.tools/build990/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/990/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build990/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build990/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build990/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build990/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build990/blogWCAG2AAhtmlcs.txt

**Comment by aaronjchristian** [comment]
Ahhh thanks @podarok i removed it from the default module and forgot to remove it from carnation. Thanks for doing that and merging this one in!

**Pull request merged**